requirements.txt

```diff
@@ -1,12 +1,12 @@
 coverage==4.5.1
 psutil==5.6.6
-requests==2.20.0
-responses==0.9.0
+requests==2.26.0
+responses~=0.16.0
 tqdm==4.19.6
 htsget==0.2.5
-urllib3==1.24.2
+urllib3==1.26.7
 
-pytest~=6.0.1
+pytest~=6.2.5
```

Comment:
pytest didn't run under Python 3.10, so I had to increase its version. This changed rippled through and necessitated the upgrade of other libraries, too.